### ISSUE TITLE
Show party emblems on constituency and person pages

### ISSUE THREAD

**Issue by andylolz** [DemocracyClub/WhoCanIVoteFor#24]
Party emblems (and with it, colours) would make constituency and person pages feel more familiar.

**Comment by jf1** [comment]
Using party descriptions would be useful too - seeing "Conservative and Unionist Party" still seems strange.

**Comment by andylolz** [comment]
> Using party descriptions would be useful too

I do agree. But unfortunately it’s a harder one to solve (and there’s a separate issue for that in YNR)

**Comment by jf1** [comment]
Indeed, didn't intend to pollute this issue - so apologies - but feel it is related, especially thinking from a user's perspective. A standard Conservative party logo would probably help mitigate the party name/description issue. Also raised in https://github.com/mysociety/yournextrepresentative/issues/823

**Comment by JoeMitchell** [comment]
Just looking at the elections/area page, I wonder if a nice, non-messy way of doing this is with a colour strip on each card...

![colouring_candidates](https://user-images.githubusercontent.com/8096305/33491139-3f52bd4e-d6b1-11e7-91f6-ec3c1da43961.jpg)

Or on the other side... 

![north_devon_local_election__the_4_candidates_in_newport___who_can_i_vote_for__by_democracy_club](https://user-images.githubusercontent.com/8096305/33492603-731a4e5e-d6b5-11e7-933d-ed1f2bf10eb1.jpg)

(There are some party colours [here](http://richardallen.co.uk/ukpolcss/), though they might have changed since then... )

We do something similar on the uk_results pages:
![image](https://user-images.githubusercontent.com/8096305/33490982-c4521a4a-d6b0-11e7-836c-c2438424ff3b.png)

**Comment by chris48s** [comment]
> There are some party colours [here](http://richardallen.co.uk/ukpolcss/)

..or [here](https://en.wikipedia.org/wiki/Wikipedia:Index_of_United_Kingdom_political_parties_meta_attributes), or [here](http://blog.richardallen.co.uk/uk-political-party-web-colours/)..

**Comment by JoeMitchell** [comment]
Oh, yes, probably the Wikipedia list is best.

**Comment by symroe** [comment]
Last time I looked, Wikipedia was wrong for every party I checked…

…and none of the lists agreed with each other…

But yes, this is a nice idea, no matter what colour the bike shed actually is :)

**Comment by JoeMitchell** [comment]
Definitely an opportunity for a Register of Party Colours here :D
